src/ApiService/ApiService/OneFuzzTypes/Model.cs

```diff
@@ -1178,3 +1178,11 @@ string ReproCmd
 public interface ITruncatable<T> {
     public T Truncate(int maxLength);
 }
+
+public record AdoNotificationEntry(
+    [PartitionKey] Guid JobId,
+    [RowKey] int Id,
```

Comment:
A work item ID needs an associated ADO org. We should store the ADO org here as well, otherwise it'll be very painful to figure out which org it was created in. Could we also name this `WorkItemId` so we know what the ID is for?